CHANGELOG.rst

```diff
@@ -220,6 +220,10 @@ time or change existing behaviors in order to make them less surprising/more use
   still present but is now considered deprecated. Thanks to `@RedBeardCode`_ and `@tomviner`_
   for the PR (`#1626`_).
 
+* optparse type usage now triggers DeprecationWarnings (`#1740`_)
```

Comment:
Please add `.` to the end of each item here.

Also, I think if you surround "optparse" withou double ` it would read better in the CHANGELOG.